components/event-block/EventBlockChapters.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div class="event-block-chapters">
+    <p class="event-block-chapters__title">
+      {{ title }}
+    </p>
+
+    <swiper :options="swiperOptions">
+      <swiper-slide
+        v-for="item in items"
+        :key="item.slug"
+      >
+        <nuxt-link
+          :to="`${currentRoute}/${item.slug}`"
```

Comment:
Don't think we need the `currentRoute`, instead we can use `append` on `nuxt-link`. This also makes it possible for this whole component to be `functional`.